extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -130,6 +139,9 @@ describes.realWin('amp-story-bookend', {
     registerServiceBuilder(win, 'localization', () => localizationService);
 
     bookend = new AmpStoryBookend(bookendElem);
+
+    // Force sync mutateElement.
+    sandbox.stub(bookend, 'mutateElement').callsArg(0);
```

Comment:
@newmuis this fixed the `this.element.getResources is not a funciton` error for me

Reply:
TIL about `callsArg`!